slam-libraries/viam-orb-slam3/orbslam_server_v1.h

```diff
@@ -119,5 +121,8 @@ int FindFrameIndex(const std::vector<std::string> &filesRGB,
 // currently does not support millisecond resolution
 string MakeFilenameWithTimestamp(string path_to_dir, string camera_name);
 
+// Removes data file
+int RemoveFile(std::string file_path);
```

Comment:
int is return to match RemoveFile previously defined in cartographer. Do we want to check and throw error if delete file fails or just ignore return?

Reply:
You can change to void, both here and in cartographer.

Reply:
Ah, sorry, I didn't notice we use the return value in tests. If we do continue to have a return here, bool would be more idiomatic for modern C++